model.py

```diff
@@ -139,7 +142,16 @@ def load_model(cls, path, cuda=False):
         model = cls(rnn_hidden_size=package['hidden_size'], nb_layers=package['hidden_layers'],
                     labels=package['labels'], audio_conf=package['audio_conf'],
                     rnn_type=supported_rnns[package['rnn_type']])
+        # the blacklist parameters are params that were previous erroneously saved by the model
+        # care should be taken in future versions that if batch_norm on the first rnn is required
+        # that it be named something else
+        blacklist = ['rnns.0.batch_norm.module.weight', 'rnns.0.batch_norm.module.bias', 'rnns.0.batch_norm.module.running_mean', 'rnns.0.batch_norm.module.running_var']
```

Comment:
I can be swayed to not include this and to instead write a script (to possibly check-in to a `utils` directory or something) to convert these old model formats.

Reply:
Won't previous models not work without these weights though? or atleast not give the same output?

Reply:
We had a bug in the previous implementation, I think, where the batch rnn was added to the model, but never actually trained or used during decode. The parameters were just registered.

See below snippets:
1. https://github.com/SeanNaren/deepspeech.pytorch/blob/522ce348259507fafe07f0eb9028d91b1fa440d1/model.py#L45 In this implementation of `BatchRNN`, even if `batch_norm=False`, the BatchNorm module is added to this module. This registers the parameters.

2. https://github.com/SeanNaren/deepspeech.pytorch/blob/522ce348259507fafe07f0eb9028d91b1fa440d1/model.py#L93 here you see that the first BatchRNN's `batch_norm` parameter is hardcoded to False.